homeassistant/components/blink/__init__.py

```diff
@@ -61,7 +61,7 @@
 
 SERVICE_SAVE_VIDEO_SCHEMA = vol.Schema({
     vol.Required(CONF_NAME): cv.string,
-    vol.Required(CONF_FILENAME): cv.string,
+    vol.Required(CONF_FILENAME): cv.template,
```

Comment:
Isn't it already possible to use templates in the service data, by using `data_template` instead of `data` in the automation?

https://www.home-assistant.io/docs/automation/templating/